layout/_partials/post/post-followme.swig

```diff
@@ -0,0 +1,23 @@
+{%- if theme.follow_me.enable %}
+  {%- set builtin = ['twitter', 'weibo', 'telegram', 'wechat'] %}
+
+  <div class="followme flex-column align-center">
+    <p>欢迎关注我的其它发布渠道</p>
```

Comment:
Thanks for your contributing!

Can you make it multilingual here? You need to add a key in `/languages/en.yml`

https://github.com/theme-next/hexo-theme-next/blob/master/languages/en.yml

Reply:
Sure, added in the second commit.